Backend/request_handler/server.py

```diff
@@ -26,11 +26,20 @@
     model_type=f"{model_name}",
 
 )
+ 
+
 def load_db():
-    with open('db/data.json', 'r') as f:
-        data = json.load(f)
-        return data
-        
+    print("Current working directory:", os.getcwd())
+    try:
+        with open('db/data.json', 'r') as f:
+            data = json.load(f)
+            # print("Data loaded:", data)
+            return data
+    except Exception as e:
+        print("Failed to load data:", e)
+
```

Comment:
Do explain the reason why this was failing to load the db and how did just putting in inside a try-catch fixed it.